src/components/Banner/Banner.jsx

```diff
@@ -136,7 +136,7 @@ export class Banner extends Component {
           visible
           full-width
           closeable={showClose}
-          onCloseAlert={onCloseAlert}
+          onClose={onCloseAlert}
```

Comment:
`close` is the event name, not `closeAlert`.

[Slack thread](https://dsva.slack.com/archives/CBU0KDSB1/p1633020309310600).